src/levanter/models/llama.py

```diff
@@ -257,6 +257,13 @@ def init(config: LlamaConfig, *, key) -> "LlamaAttention":
         )
         return LlamaAttention(config, q_proj, k_proj, v_proj, o_proj)
 
+    def _rope_scale_factor(self) -> float:
```

Comment:
Feel free to file an issue and I can look at normalizing how Gemma handles this. I can't remember if HFGemma doesn't use it at all or if it was doing something different and weird.

Reply:
HfGemma doesn't use it at all, which seems like an oversight more than anything. (Llama 2 also doesn't use it by default, but you can turn it on and do some tuning to get the length extrapolation, I think)